plugin/src/main/java/org/opennms/velocloud/requisition/AbstractRequisitionProvider.java

```diff
@@ -44,7 +44,9 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 public abstract class AbstractRequisitionProvider<Req extends AbstractRequisitionProvider.Request> implements RequisitionProvider {
-    private static final Logger LOG = LoggerFactory.getLogger(AbstractRequisitionProvider.class);
+    protected static final Logger LOG = LoggerFactory.getLogger(AbstractRequisitionProvider.class);
+
+    public static final String VELOCLOUD_CONTEXT = "_Velocloud_";
```

Comment:
```suggestion
    public static final String VELOCLOUD_METADATA_CONTEXT = "velocloud";
```

We can reserve this context to internal use. See https://docs.opennms.com/horizon/30/operation/meta-data.html?q=metadata#ga-metadata-contexts.